code/site/api.php

```diff
@@ -1,59 +1,63 @@
- * @link http://techjoomla.com
+ * @package     Joomla.Site
+ * @subpackage  Com_api
+ *
+ * @copyright   Copyright (C) 2009-2014 Techjoomla, Tekdi Technologies Pvt. Ltd. All rights reserved.
+ * @license     GNU GPLv2 <http://www.gnu.org/licenses/old-licenses/gpl-2.0.html>
+ * @link        http://techjoomla.com
  * Work derived from the original RESTful API by Techjoomla (https://github.com/techjoomla/Joomla-REST-API)
  * and the com_api extension by Brian Edgerton (http://www.edgewebworks.com)
-*/
-
-defined( '_JEXEC' ) or die( 'Restricted access' );
+ */
+defined('_JEXEC') or die('Restricted access');
 
 jimport('joomla.application.component.controller');
 
-$library_path = JPATH_COMPONENT.'/libraries';
-
-JLoader::register('APIController', $library_path.'/controller.php');
-JLoader::register('APIModel', $library_path.'/model.php');
-JLoader::register('APIView', $library_path.'/view.php');
-JLoader::register('APIPlugin', $library_path.'/plugin.php');
-JLoader::register('APIError', $library_path.'/error.php');
-JLoader::register('ApiException', $library_path.'/exception.php');
-JLoader::register('APICache', $library_path.'/cache.php');
-JLoader::register('APIResource', $library_path.'/resource.php');
-JLoader::register('APIAuthentication', $library_path.'/authentication.php');
-JLoader::register('APIAuthenticationKey', $library_path.'/authentication/key.php');
-JLoader::register('APIAuthenticationLogin', $library_path.'/authentication/login.php');
-JLoader::register('APIAuthenticationSession', $library_path.'/authentication/session.php');
-JLoader::register('APIHelper', $library_path.'/helper.php');
-JTable::addIncludePath(JPATH_ROOT.'/administrator/components/com_api/tables');
+$library_path = JPATH_COMPONENT . '/libraries';
+
+JLoader::register('APIController', $library_path . '/controller.php');
+JLoader::register('APIModel', $library_path . '/model.php');
+JLoader::register('APIView', $library_path . '/view.php');
+JLoader::register('APIPlugin', $library_path . '/plugin.php');
+JLoader::register('APIError', $library_path . '/error.php');
+JLoader::register('ApiException', $library_path . '/exception.php');
+JLoader::register('APICache', $library_path . '/cache.php');
+JLoader::register('APIResource', $library_path . '/resource.php');
+JLoader::register('APIAuthentication', $library_path . '/authentication.php');
+JLoader::register('APIAuthenticationKey', $library_path . '/authentication/key.php');
+JLoader::register('APIAuthenticationLogin', $library_path . '/authentication/login.php');
+JLoader::register('APIAuthenticationSession', $library_path . '/authentication/session.php');
+JLoader::register('APIHelper', $library_path . '/helper.php');
+JTable::addIncludePath(JPATH_ROOT . '/administrator/components/com_api/tables');
 
 $app = JFactory::getApplication();
 
-//$view	= JRequest::getCmd('view', '');
-$view = $app->input->get('view','','CMD');
+$view = $app->input->get('view', '', 'CMD');
```

Comment:
CMD is already the default, so you can remove the third argument. Refer https://api.joomla.org/cms-3/classes/JInput.html#method_get